scripts/generateProptypes.ts

```diff
@@ -202,6 +202,19 @@ async function generateProptypes(
       const ignoreGenerated =
         previous !== undefined &&
         previous.startsWith('PropTypes /* @typescript-to-proptypes-ignore */');
+
+      if (
+        ignoreGenerated &&
+        // `ignoreGenerated` implies that `previous !== undefined`
+        previous!
+          .replace('PropTypes /* @typescript-to-proptypes-ignore */', 'PropTypes')
+          .replace(/\s/g, '') === generated.replace(/\s/g, '')
```

Comment:
Works for now even though whitespace does add semantics. Hopefully we don't have to refine this later. If it gets out of hand we can always downgrade it to a CI-only warning.